linkis-commons/linkis-module/src/main/scala/org/apache/linkis/server/conf/Knife4jConfig.scala

```diff
@@ -0,0 +1,70 @@
+package org.apache.linkis.server.conf
```

Comment:
Missing  ASF header

Reply:
I will add it and submit another pull request